test/e2e/data/infrastructure-docker/main/clusterclass-quick-start.yaml

```diff
@@ -20,10 +20,7 @@ spec:
         name: quick-start-control-plane
     machineHealthCheck:
       maxUnhealthy: 100%
-      unhealthyConditions:
-      - type: e2e.remediation.condition
-        status: "False"
-        timeout: 20s
+      # We are intentionally not setting the 'unhealthyConditions' here to test that the field is optional.
```

Comment:
To test both scenarios, let's keep unhealthy conditions for one of CP/MD (and drop only for the other)

Reply:
I change it